binlog_streamer.go

```diff
@@ -153,19 +154,22 @@ func (s *BinlogStreamer) Run() {
 		case *replication.RotateEvent:
 			// This event is needed because we need to update the last successful
 			// binlog position.
+			s.lastResumableBinlogPosition.Pos = uint32(e.Position)
```

Comment:
Shouldn't we need to set the `.Name` here as well? This is a RotateEvent so the file should change, right? That would get rid of the slightly cryptic `s.lastResumableBinlogPosition.Name` line below?

Reply:
Following up on our conversation, we decided to set the name here and in the `ConnectBinlogStreamerToMysqlFrom` above 👍